stixcore/processing/tests/test_processing.py

```diff
@@ -268,11 +270,69 @@ def test_single_vs_batch(out_dir):
         CONFIG.set('Logging', 'stop_on_error', str(CONTINUE_ON_ERROR))
 
 
-if __name__ == '__main__':
-    '''TO BE REMOVED
+def test_pipeline_logging(spicekernelmanager, out_dir):
+
+    CONTINUE_ON_ERROR = CONFIG.getboolean('Logging', 'stop_on_error', fallback=False)
+    FITS_ARCHIVE = CONFIG.get('Paths', 'fits_archive')
+    LOG_LEVEL = CONFIG.get('Pipeline', 'log_level')
+    LOG_DIR = CONFIG.get('Pipeline', 'log_dir')
+    try:
+        CONFIG.set('Logging', 'stop_on_error', str(False))
+        CONFIG.set('Paths', 'fits_archive', str(out_dir / "fits"))
+        CONFIG.set('Pipeline', 'log_level', str('DEBUG'))
+        CONFIG.set('Pipeline', 'log_dir', str(out_dir / "logging"))
+
+        log_dir = Path(CONFIG.get('Pipeline', 'log_dir'))
+        log_dir.mkdir(parents=True, exist_ok=True)
+
+        for f in test_data.tmtc.XML_TM:
+            process_tm(f, spm=spicekernelmanager)
+
+        assert len(list(log_dir.rglob("*.log"))) == 3
+        assert len(list(log_dir.rglob("*.log.err"))) == 0
+        assert len(list(log_dir.rglob("*.out"))) == 3
+        # TODO increase if level2 for more products is available
+        assert len(list(Path(CONFIG.get('Paths', 'fits_archive')).rglob("*.fits"))) == 11
+
+    finally:
+        CONFIG.set('Logging', 'stop_on_error', str(CONTINUE_ON_ERROR))
+        CONFIG.set('Paths', 'fits_archive', str(FITS_ARCHIVE))
+        CONFIG.set('Pipeline', 'log_level', str(LOG_LEVEL))
+        CONFIG.set('Pipeline', 'log_dir', str(LOG_DIR))
 
-    currently used to start the AUX processing on an existing L1 fits dir
-    '''
+
+@pytest.mark.skipif(not CONFIG.getboolean('Pipeline', 'error_mail_send', fallback=False),
```

Comment:
As mentioned above you could mock this so the test could run

Reply:
good point will do so